### DISCUSSION TITLE
Change logging level of apscheduler

### DISCUSSION THREAD

**Discussion by sagrawal8** [agronholm/apscheduler#597] [Q&A]
Is there anyway to have apscheduler print logs at DEBUG level instead of INFO level. The other program also prints logs at INFO level and when sorting through the logs using their level, I see a lot of apscheduler logs that I dont want to see.

<img width="1434" alt="image" src="https://user-images.githubusercontent.com/61482309/155609539-d8a63002-fe0b-4557-bbf9-6df2b0f99364.png">

**Comment by agronholm** [comment]
This is essentially the same question as #595. Please see the provided link there for a guide on how to configure the logging system to only print what you're interested in.

**Reply by sagrawal8** [reply]
Not quite. I believe my issue is different. My issue was more related to me still being able to see the logs but the log message itself is at the debug level.

**Reply by agronholm** [reply]
So is something preventing you from configuring the `apscheduler` loggers to only show `WARNING` and higher level messages?
